cita-executor/src/postman.rs

```diff
@@ -357,80 +364,78 @@ impl Postman {
-        //             .map(|receipt| receipt.transaction_hash)
-        //             .filter(|hash| hash != &H256::default())
-        //             .collect();
-
-        //         let schedule = Schedule::default();
-        //         // Filter out accounts in the black list where the account balance has reached the benchmark value.
-        //         // Get the smaller value between tx_create_gas and tx_gas for the benchmark value.
-        //         let bm_value = std::cmp::min(schedule.tx_gas, schedule.tx_create_gas);
-        //         let mut clear_list: Vec<Address> = self
-        //             .black_list_cache
-        //             .read()
-        //             .unwrap()
-        //             .values()
-        //             .filter(|address| {
-        //                 close_block
-        //                     .state
-        //                     .balance(address)
-        //                     .and_then(|x| Ok(x >= U256::from(bm_value)))
-        //                     .unwrap_or(false)
-        //             })
-        //             .cloned()
-        //             .collect();
-
-        //         // Get address of sending account by transaction hash
-        //         let blacklist: Vec<Address> = close_block
-        //             .body()
-        //             .transactions()
-        //             .iter()
-        //             .filter(|tx| blacklist_transaction_hash.contains(&tx.get_transaction_hash()))
-        //             .map(|tx| *tx.sender())
-        //             .collect();
-
-        //         {
-        //             let mut black_list_cache = self.black_list_cache.write().unwrap();
-        //             black_list_cache
-        //                 .prune(&clear_list)
-        //                 .extend(&blacklist[..], close_block.number());
-        //             clear_list.extend(black_list_cache.lru().iter());
-        //         }
-
-        //         let black_list = BlackList::new()
-        //             .set_black_list(blacklist)
-        //             .set_clear_list(clear_list);
-
-        //         if !black_list.is_empty() {
-        //             let black_list_bytes: Message = black_list.protobuf().into();
-
-        //             info!(
-        //                 "black list is {:?}, clear list is {:?}",
-        //                 black_list.black_list(),
-        //                 black_list.clear_list()
-        //             );
-
-        //             self.response_mq(
-        //                 routing_key!(Executor >> BlackList).into(),
-        //                 black_list_bytes.try_into().unwrap(),
-        //             );
-        //         }
-        //     }
-        //     EconomicalModel::Quota => {}
-        // }
+    //       this opinion in the future.
+    // To Be reconisder black list
+    fn pub_black_list(&self, close_block: &ClosedBlock) {
+        // Get all transaction hash that is reported as not enough quota
```

Comment:
No considering economical?

Reply:
economical maybe not send,from the source